cmd/cat-main.go

```diff
@@ -102,6 +102,13 @@ EXAMPLES:
 `,
 }
 
+// checkCatSyntax - validate all the passed arguments
+func checkCatSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) < 1 || len(ctx.Args()) > 2 {
```

Comment:
@r-scheele I think 'mc cat' supports multiple arguments, like `mc cat arg1 arg2 arg3 arg4, ..`

can you double check ?

Reply:
You're right. My bad.